app/views/admin/loans/_transactions.html.slim

```diff
@@ -0,0 +1,14 @@
+section.transactions
+  .show-actions.hidden-print
+    a.btn.btn-primary data-action="new-transaction" href="#" = t(".new_transaction")
+
+  = grid(@transactions_grid) do |g|
+    - g.column name: 'Id', attribute: 'qb_id'
+    - g.column name: 'Type' do |transaction|
+      - transaction.qb_transaction_type
+    - g.column name: 'Date' do |transaction|
+      - transaction.txn_date
+    - g.column name: 'Total' do |transaction|
+      - transaction.total
+    - g.column name: 'Note' do |transaction|
```

Comment:
I don't think wice_grid respects I18N for column names. I took a quick look at the docs and source, this appears to be the case.